react-payments-app/src/components/Input/input.css

```
@@ -34,7 +43,12 @@
   border: none;
   border-radius: 0.25rem;
 }
-
+.input-of-pw {
+  background: none;
+}
+.input-bigger-text {
+  font-size: 150%;
+}
```

```suggestion

.input-of-pw {
  background: none;
}

.input-bigger-text {
  font-size: 150%;
}

```

클래스와 클래스 사이는 개행문자를 넣어주면 가독성이 좋아집니다 😄